app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'CMSFilterApp'
+
+app = Chalice(app_name=app_name)
+
+# Initialize the logger
+logging.basicConfig(filename='app.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(app_name)
+
+# Set the log level from Environment variable LOG_LEVEL
+if os.environ.get('LOG_LEVEL') is None:
+    logger.setLevel(logging.DEBUG)
+else:
+    logger.setLevel(os.environ['LOG_LEVEL'])
+
+kinesis_endpoint_url = os.environ.get('KINESIS_ENDPOINT_URL')
+aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID', 'iam')
+aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY', 'iam')
+aws_region = os.environ.get('AWS_REGION', 'us-east-1')
+output_stream_name = os.environ.get('OUTPUT_STREAM')
+logger.debug('##Environment variables {}'.format(os.environ))
+
+if os.environ['OUTPUT_STREAM'] is None:
+    raise KeyError('Output stream is required before running the lambda!!')
```

Comment:
is that actually the case? I thought the lambda return values are usually send to a stream fi the lambda is configured to send data to a stream.

Reply:
I haven't tested that, but when I was doing it with local-stack I have to pass the kinesis output stream name.

Reply:
yes, I checked that and its only the same stream name that can be passed as the output/destination. and also I checked on google and found this way only to insert data from lambda in a diff kineiss stream.